packages/amino/src/multisig.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { compareArrays, createMultisigThresholdPubkey } from "./multisig";
+import { test1, test2, test3, testgroup1, testgroup2, testgroup3, testgroup4 } from "./testutils.spec";
+
+describe("multisig", () => {
+  describe("compareArrays", () => {
+    it("return 0 for equal arrays", () => {
+      expect(compareArrays(new Uint8Array([]), new Uint8Array([]))).toEqual(0);
+      expect(compareArrays(new Uint8Array([1]), new Uint8Array([1]))).toEqual(0);
+      expect(compareArrays(new Uint8Array([3, 2, 1]), new Uint8Array([3, 2, 1]))).toEqual(0);
+    });
+
+    it("return > 0 for left > right", () => {
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([5, 5, 4]))).toBeGreaterThan(0);
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([5, 4, 5]))).toBeGreaterThan(0);
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([4, 5, 5]))).toBeGreaterThan(0);
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([5, 5]))).toBeGreaterThan(0);
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([5]))).toBeGreaterThan(0);
+      expect(compareArrays(new Uint8Array([5, 5, 5]), new Uint8Array([]))).toBeGreaterThan(0);
+    });
+
+    it("return < 0 for left < right", () => {
+      expect(compareArrays(new Uint8Array([5, 5, 4]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+      expect(compareArrays(new Uint8Array([5, 4, 5]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+      expect(compareArrays(new Uint8Array([4, 5, 5]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+      expect(compareArrays(new Uint8Array([5, 5]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+      expect(compareArrays(new Uint8Array([5]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+      expect(compareArrays(new Uint8Array([]), new Uint8Array([5, 5, 5]))).toBeLessThan(0);
+    });
```

Comment:
Would be good to have cases like `[4,5,5]` vs `[5,5,4]`, ie is it left or right precedence? Also `[6]` vs `[5,5]`, ie is magnitude or length more important?